ui/component/upcomingClaims/index.js

```diff
@@ -66,13 +67,13 @@ const select = (state, props) => {
     scheduledOptions,
     livestreamUris: csByQuery[loKey],
     scheduledUris: csByQuery[soKey],
+    hideUpcoming: selectClientSetting(state, SETTINGS.HIDE_SCHEDULED_LIVESTREAMS),
```

Comment:
`SETTINGS.HIDE_SCHEDULED_LIVESTREAMS` is now "hide in Following Page", right?  For the homepage, the visibility is governed by the homepage-order setting?

(Just to confirm)

Reply:
Yeah, I think that's fine, because it touches the wallet and would end up leaving 2 states there, so I didn't suggest renaming it.  Can't recall it if does a replace, merge or addition.

Reply:
Aight...and yeah, it's now the hide state on the following page.